### DISCUSSION TITLE
EditableGeoJsonLayer not able to change color with a state variable

### DISCUSSION THREAD

**Discussion by even996** [visgl/deck.gl#5916] [Q&A]
Hi!
I want to use nebula's `EditableGeoJsonLayer `and to update the colors for the layer.  I'm using a `boolean `state `variable` to update the colors with the `getLineColor` attribute and `updateTriggers`. 
This approach works for the deck gl `GeoJsonLaye ` with `updateTriggers` and is not working for EditableGeoJsonLayer.

Thanks for helping  :)

Code --> https://codesandbox.io/s/interesting-thompson-e8ntf?file=/src/App.js

**Comment by Pessimistress** [comment]
`EditableGeoJsonLayer` is not part of deck.gl. You should ask in the nebula.gl repo.
